adafruit_bno055.py

```diff
@@ -244,3 +222,58 @@ def use_external_crystal(self, value):
         self._write_register(_TRIGGER_REGISTER, 0x80 if value else 0x00)
         self.mode = last_mode
         time.sleep(0.01)
+
+
+    @property
+    def temperature(self):
+        """Measures the temperature of the chip in degrees Celsius."""
+        return self._temperature
+
+    @property
+    def acceleration(self):
+        """Gives the raw accelerometer readings, in m/s."""
+        if self.mode not in [0, 2, 3, 6]:
+            return self._acceleration
+        return (None, None, None)
```

Comment:
I would also update the docstrings of the mode property and the new properties that may return `None` to describe the new behavior.

Reply:
The docstrings in the individual properties are great. All I would do for the mode property is to add a legend at the top of the table to describe what each symbol in the table means  (x=on, -=off and will return an empty tuple)